source/deploy/mobile-app-changelog.md

```diff
@@ -2,12 +2,38 @@
 
 Latest Mattermost Mobile Apps releases:
 
-- [1.55.1 Release](#release)
-- [1.55.0 Release](#id1)
-- [1.54.0 Release](#id5)
-- [1.53.0 Release](#id10)
-- [1.52.0 Release](#id14)
-- [1.51.2 Release](#id18)
+- [2.0.0 Release](#release)
+- [1.55.1 Release](#id1)
+- [1.55.0 Release](#id4)
+- [1.54.0 Release](#id8)
+- [1.53.0 Release](#id13)
+- [1.52.0 Release](#id17)
+
+## 2.0.0 Release
+- Release Date: December 15, 2022
+- Server Versions Supported: Server v7.1.0+ is required. Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device.
+
+### Compatibility
+ - **Upgrade to server version v7.1.0 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/upgrade/extended-support-release.html) (ESR) 6.3.0 has ended and upgrading to server ESR v7.1.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 12.1+ is required.
+
+### Highlights
+
+#### Multiple Workspaces
```

Comment:
@amyblais propose we replace the highlights section with the first few paragraphs of this blog: https://mattermost.com/blog/preparing-for-mobile-v2-0/

Lets also link to that blog for more details in the compatibility section